src/main/java/net/minecraftforge/fluids/BlockFluidBase.java

```diff
@@ -728,4 +728,21 @@ public AxisAlignedBB getCollisionBoundingBox(@Nonnull IBlockState blockState, @N
     {
         return NULL_AABB;
     }
+    
+    @SideOnly (Side.CLIENT)
+    public Vec3d getFogColor(World world, BlockPos pos, IBlockState state, Entity entity, Vec3d originalColor, float partialTicks)
+    {
+        if (getFluid() != null)
+        {
+            int color = getFluid().getColor();
+            int red = color >> 16 & 0xFF;
+            int green = color >> 8 & 0xFF;
+            int blue = color & 0xFF;
+            return new Vec3d(red, green, blue);
```

Comment:
This will not work, you need to ``` / 255``` each value. getFogColor must return 0 - 1 color values